### PR TITLE
Update links to bug trackers to point to GitHub

### PR DESCRIPTION
**Pull request by cipherboy**
`Signed-off-by: Alexander Scheel <ascheel@redhat.com>`

---

As reported by @fcami.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Therefore we would love any input you would consent to share! Any and all contri
 
 ## We Develop with Github
 
-We use github to host code and feature requests, as well as accept pull requests. We use [Pagure](https://pagure.io/dogtagpki/issues) to track issues (we will be moving to github issues soon).
+We use github to host code and feature requests, as well as accept pull requests. We use [GitHub](https://github.com/dogtagpki/pki/issues) to track issues (we will be moving to github issues soon).
```
Comment (SilleBille) [inline CONTRIBUTING.md:16]: ~(we will be moving to github issues soon)~

We are in github issues now ;-)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ See also [PKI specific github cheatsheet](https://www.dogtagpki.org/wiki/GitHub_
 
 ## Report Issues
 
-We use Pagure issues to track public bugs. Report a bug by [opening a new issue](https://pagure.io/dogtagpki/new_issue); it's that easy!
+We use Pagure issues to track public bugs. Report a bug by [opening a new issue](https://github.com/dogtagpki/pki/issues/new/choose); it's that easy!
```
Comment (SilleBille) [inline CONTRIBUTING.md:32]: We use **GitHub** issues to track public bugs

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ See also [PKI specific github cheatsheet](https://www.dogtagpki.org/wiki/GitHub_
...
 
 **Note:** *We will be moving to Github Issues soon. All existing Pagure issues will be migrated.*
```
Comment (SilleBille) [inline CONTRIBUTING.md:34]: This is done. We can remove this line :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ We use Pagure issues to track public bugs. Report a bug by [opening a new issue]
 
 **Notes:** possibly include theories to explain the behaviour, or steps you tried that didn't work or any other information you think that’s valuable that will help our team to provide a fix soon.
 
-People ***love*** thorough bug reports. [This is a good example](https://pagure.io/dogtagpki/issue/3194) of a thorough bug report.
+People ***love*** thorough bug reports. [This is a good example](https://github.com/dogtagpki/pki/issue/3311) of a thorough bug report.
```
Comment (SilleBille) [inline CONTRIBUTING.md:50]: The link to GH is broken. It should be `https://github.com/dogtagpki/pki/issues/3311`

**Review by SilleBille — COMMENTED** [review]
There are some still traces of Pagure

File: CONTRIBUTING.md (hunk 4 above)
Comment (cipherboy) [reply CONTRIBUTING.md]: Ah right. It is `/pull/<id>` but `/issues/<id>`...

**Comment by cipherboy** [comment]
@SilleBille -- updated!

**Review by SilleBille — APPROVED** [review]
Looks clean! Thanks @cipherboy :+1:

**Review by edewata — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**